ui/autocomplete.js

```diff
@@ -365,7 +365,7 @@ $.widget( "ui.autocomplete", {
 		}
 
 		if ( !element || !element[ 0 ] ) {
-			element = this.element.closest( ".ui-front" );
+			element = this.element.closest( ".ui-front,dialog" );
```

Comment:
It's a habit that creeps in every once in a while. I blame John for always using no spaces in selectors since they can't be munged by minification tools. Off the top of my head, I was able to think of two places we have selectors with commas and one has spaces and one doesn't :-P

I can add the spaces in; I actually prefer it that way.